lib/config/config.js

```diff
@@ -56,6 +56,14 @@ const {
   normalizeSpanStackTraceMinDuration
 } = require('./normalizers')
 
+let threadId
+try {
```

Comment:
nit: Both `isMainThread` & `threadId` properties were added in v10.5.0 so what's your opinion on checking on the version

```javascript
const threadsSupported = semver.gte(process.version, '10.5.0')
let threadId
if (threadsSupported) {
  threadId = require('worker_threads').threadId
}
```

if you prefer the try/catch approach I would drop a comment telling why it may throw